package.json

```diff
@@ -95,4 +99,4 @@
     "xmldom": "^0.2.1",
     "yargs": "^15.0.2"
   }
-}
+}
```

Comment:
Trivial: @ButterB0wl NEWLINE

Reply:
idk what happened there vscode just did it.  Should there be a newline or did i inadvertently add a new line

Reply:
Generally add a newline, always.